modules/elasticsearch-impl/src/main/java/org/opencastproject/elasticsearch/impl/AbstractElasticsearchIndex.java

```diff
@@ -414,6 +426,17 @@ protected void init(int version)
       client = new RestHighLevelClient(builder);
     }
 
+    String indexServiceUrl = getIndexServiceUrl();
+    while (!isIndexServiceReachable(indexServiceUrl)) {
+      logger.warn("Could not reach Elasticsearch at {}. Trying again after few seconds...", indexServiceUrl);
```

Comment:
Since we want people to use OIpenSearch?
```suggestion
      logger.warn("Could not reach OpenSearch at {}. Trying again after few seconds...", indexServiceUrl);
```